code/game/gamemodes/clock_cult/clock_mobs.dm

```diff
@@ -0,0 +1,214 @@
+/mob/living/simple_animal/hostile/anima_fragment //Anima fragment: Low health but high melee power. Created by inserting a soul vessel into an empty fragment.
+	name = "anima fragment"
+	desc = "An ominous humanoid shell with a single glowing yellow eye, pushed into the air by jets of blazing yellow fire."
+	icon = 'icons/mob/clockwork_mobs.dmi'
+	icon_state = "anima_fragment"
+	health = 75 //Glass cannon
+	maxHealth = 75
+	wander = FALSE
+	minbodytemp = 0
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0) //Robotic
+	healable = FALSE
+	melee_damage_lower = 25
+	melee_damage_upper = 25
+	attacktext = "crushes"
+	attack_sound = 'sound/magic/clockwork/anima_fragment_attack.ogg'
+	var/playstyle_string = "<span class='heavy_brass'>You are an anima fragment</span><b>, a clockwork creation of Ratvar. As a fragment, you are weak but possess powerful melee capabilities \
+	in addition to being immune to extreme temperatures and pressures. Your goal is to serve the Justiciar and his servants in any way you can. You yourself are one of these servants, and will \
+	be able to utilize anything they can, assuming it doesn't require opposable thumbs.</b>"
+
+/mob/living/simple_animal/hostile/anima_fragment/death(gibbed)
+	..(TRUE)
+	visible_message("<span class='warning'>[src]'s flame jets cut out as it falls to the floor with a tremendous crash. A cube of metal tumbles out, whirring and sputtering.</span>", \
+	"<span class='userdanger'>Your gears seize up. Your flame jets flicker. Your soul vessel belches smoke as you helplessly crash down.</span>")
+	playsound(src, 'sound/magic/clockwork/anima_fragment_death.ogg', 100, 1)
+	new/obj/item/clockwork/component/replicant_alloy/smashed_anima_fragment(get_turf(src))
+	new/obj/item/device/mmi/posibrain/soul_vessel(get_turf(src)) //Notice the lack of transfer - it's a standard soul vessel with no mind in it!
+	qdel(src)
+	return 1
+
+/mob/living/simple_animal/hostile/clockwork_marauder //Clockwork marauder: Slow but with high damage, resides inside of a servant. Created via the Memory Allocation scripture.
```

Comment:
Frankly it'd be better to have these as heavily overridden guardians, because they copy a LOT of what guardians do.